apps/web/src/app/events/components/TimeLocationBox/TimeLocationBox.tsx

```diff
@@ -26,13 +26,9 @@ export const TimeLocationBox: FC<Props> = ({
   eventDescription,
   locationLink,
 }) => {
-  const [weekday, date, time] = formatDate(datetimeStart, {
-    forceAbsolute: true,
-    includeWeekday: true,
-    includeTime: true,
-  })
-    .replace(",", "")
-    .split(" ")
+  const weekday = Intl.DateTimeFormat("nb-NO", IntlFormats.Weekday).format(datetimeStart)
+  const date = Intl.DateTimeFormat("nb-NO", IntlFormats.Date).format(datetimeStart)
+  const time = Intl.DateTimeFormat("nb-NO", IntlFormats.Time).format(datetimeStart)
```

Comment:
prefer new Intl.DateTimeFormat? It's a constructor but it accepts being used as a function